README.md

```diff
@@ -1,2 +1,21 @@
 # transformAddonDataToViews
-Transforms data so it can be accessed in nvaccess/addonDataViews
+Code to transform data so it can be accessed in nvaccess/addon-store-submission:views
```

Comment:
Please add more details about what the project aims to achieve and how it fits into the bigger system. This will help to ensure we are all on the same page about what you intend to build here.